doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,401 @@
+
+wget https://github.com/IQSS/dataverse/releases/download/v6.2/biomedical.tsv
+
+curl http://localhost:8080/api/admin/datasetfield/load -H "Content-type: text/tab-separated-values" -X POST --upload-file scripts/api/data/metadatablocks/biomedical.tsv
+```
+
+8\. For installations with custom or experimental metadata blocks:
+
+- Stop Solr instance (usually `service solr stop`, depending on Solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/6.2/installation/prerequisites.html#solr-init-script))
+
+- Run the `update-fields.sh` script that we supply, as in the example below (modify the command lines as needed to reflect the correct path of your solr installation):
+```
+	wget https://raw.githubusercontent.com/IQSS/dataverse/master/conf/solr/9.3.0/update-fields.sh
+	chmod +x update-fields.sh
+	curl "http://localhost:8080/api/admin/index/solr/schema" | ./update-fields.sh /usr/local/solr/solr-9.3.0/server/solr/collection1/conf/schema.xml
+```
+     
+- Restart Solr instance (usually `service solr restart` depending on solr/OS)
+
+9\. Reindex Solr:
+
+   For details, see https://guides.dataverse.org/en/6.0/admin/solr-search-index.html but here is the reindex command:
+
+```
+   curl http://localhost:8080/api/admin/index
+```
+
+***
+
+## ⚙️ New Settings 
+
+### Microprofile settings
+
+*The * indicates a provider id indicating which provider the setting is for*
+
+> - dataverse.pid.providers
+> - dataverse.pid.default-provider
+> - dataverse.pid.*.type
+> - dataverse.pid.*.label
+> - dataverse.pid.*.authority
+> - dataverse.pid.*.shoulder
+> - dataverse.pid.*.identifier-generation-style
+> - dataverse.pid.*.datafile-pid-format
+> - dataverse.pid.*.managed-list
+> - dataverse.pid.*.excluded-list
+> - dataverse.pid.*.datacite.mds-api-url
+> - dataverse.pid.*.datacite.rest-api-url
+> - dataverse.pid.*.datacite.username
+> - dataverse.pid.*.datacite.password
+> - dataverse.pid.*.ezid.api-url
+> - dataverse.pid.*.ezid.username
+> - dataverse.pid.*.ezid.password
+> - dataverse.pid.*.permalink.base-url
+> - dataverse.pid.*.permalink.separator
+> - dataverse.pid.*.handlenet.index
+> - dataverse.pid.*.handlenet.independent-service
+> - dataverse.pid.*.handlenet.auth-handle
+> - dataverse.pid.*.handlenet.key.path
+> - dataverse.pid.*.handlenet.key.passphrase
+> - dataverse.spi.pidproviders.directory
+> - dataverse.solr.concurrency.max-async-indexes
+
+[⬅️ Go back](#multiple-pid-sup)
+
+## SMTP Settings: 
+
+> - dataverse.mail.system-email
+> - dataverse.mail.mta.host
+> - dataverse.mail.mta.port
+> - dataverse.mail.mta.ssl.enable
+> - dataverse.mail.mta.auth
+> - dataverse.mail.mta.user
+> - dataverse.mail.mta.password
+> - dataverse.mail.mta.allow-utf8-addresses
+> - Plus many more for advanced usage and special provider requirements. See [configuration guide for a full list](https://guides.dataverse.org/en/6.2/installation/config.html#dataverse-mail-mta).
+
+[⬅️ Go back](#simplified-smtp-configuration)
```

Comment:
Can we list the new database settings as well?

Reply:
What would be the settings to list here? I couldn't find something on the note and was not sure what to grab from the PR for this. 😄

Reply:
You can see "New JVM Options and DB Settings" at https://github.com/IQSS/dataverse/releases/tag/v5.10 for an example of a release where both JVM/MPCONFIG and database settings were added.